### PR TITLE
Gui 1725 show stack template in a modal

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-1725

tested in Chrome and FF (latest) on OS X

### CODE REVIEW

**Comment by kamalgill** [comment]
Hal, retest this please

File: eucaconsole/static/js/pages/stack.js [hunk 1]
```diff
@@ -112,12 +115,39 @@ angular.module('StackPage', ['MagicSearch', 'EucaConsoleUtils'])
...
                 var results = oData ? oData.results : '';
-                $scope.templateLoading = false;
-                if (results) {
-                    $scope.description = results.description;
-                    $scope.parameters = results.parameters;
+                $scope.stackTemplate = eucaUnescapeJson(results);
+                $scope.codeEditor.setValue($scope.stackTemplate);
+                $scope.codeEditor.focus();
+            }).error(function (oData, status) {
+                var errorMsg = oData.message || '';
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/stack.js]: @dkavanagh: We should use the eucaHandleError util method here

Comment (dkavanagh) [inline eucaconsole/static/js/pages/stack.js]: copied from elsewhere.. will fix both places.

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by dkavanagh** [comment]
This is only failing on a verification of volume attach:
http://10.111.4.1/job/user-console-test-volumes/3701//artifact/VOLUME_OPERATIONS_REPORT.html

**Pull request merged**